hoprd/rest-api/src/lib.rs

```diff
@@ -816,12 +814,21 @@ mod account {
         currency: BalanceType,
         #[serde_as(as = "DisplayFromStr")]
         #[schema(value_type = String)]
-        amount: U256,
+        amount: String,
         #[serde_as(as = "DisplayFromStr")]
         #[schema(value_type = String)]
         address: Address,
     }
 
+    #[derive(Debug, Default, Clone, serde::Serialize, serde::Deserialize, utoipa::ToSchema)]
+    #[schema(example = json!({
+            "receipt": "0xb4ce7e6e36ac8b01a974725d5ba730af2b156fbe",
+    }))]
+    #[serde(rename_all = "camelCase")]
+    pub(crate) struct WithdrawResponse {
+        receipt: String,
```

Comment:
```suggestion
        #[serde_as(as = "DisplayFromStr")]
        #[schema(value_type = String)]
        receipt: Hash,
```

Reply:
applied + push additional modifications to make it work.